tests/legacy.rs

```diff
@@ -660,16 +684,26 @@ fn cube_complex_test() -> Result {
         }),
         ..out1.clone()
     };
+    let out2_bin = Vtk {
+        data: DataSet::inline(PolyDataPiece {
+            points: points.clone(),
+            polys: polys.clone(),
+            verts: verts.clone(),
+            data: attributes_bin.clone(),
+            ..Default::default()
+        }),
+        ..out1.clone()
+    };
 
     test!(parse_ne(in1) => ne(&out1));
     test_b!(parse_ne(String::new().write_vtk_ne(out1.clone())?.as_bytes()) => ne(&out1));
-    test_b!(parse_ne(Vec::<u8>::new().write_vtk_ne(out1.clone())?) => ne(&out1));
-    test_b!(parse_le(Vec::<u8>::new().write_vtk_le(out1.clone())?) => le(&out1));
-    test_b!(parse_be(Vec::<u8>::new().write_vtk_be(out1.clone())?) => out1);
+    test_b!(parse_ne(Vec::<u8>::new().write_vtk_ne(out1_bin.clone())?) => ne(&out1_bin));
```

Comment:
I think it would make sense to add two new tests here that write the binary input (`out1_bin`, `out2_bin`) into a string and see if that conversion works correctly. Should be easy enough, right?

Reply:
makes sense, I will add one